documentation/src/main/wiki/Server.md

```diff
@@ -234,3 +235,37 @@ Other option is to provide those parameters through JVM properties:
 - `-Dio.knotx.KnotxServer.options.config.serverOptions.keyStoreOptions.path=/path/to/keystore.jks` 
 - `-Dio.knotx.KnotxServer.options.config.serverOptions.keyStoreOptions.password=keyPass`
 - `-Dio.knotx.KnotxServer.options.config.serverOptions.ssl=true`
+
+### Vert.x Event Bus delivery options
+
+While HTTP request processing, Server calls other modules like Repository Connectors, Knots using 
+[Vert.x Event Bus](http://vertx.io/docs/apidocs/io/vertx/core/eventbus/EventBus.html). 
+The `config` field can contain [Vert.x Delivery Options](http://vertx.io/docs/apidocs/io/vertx/core/eventbus/DeliveryOptions.html)
+related to the event bus. It can be used to control the low level aspects of the event bus communication like timeouts, 
+headers, message codec names.
+
+
+The `repositoryDeliveryOptions` need to be added in the following place, of the KnotxServerVerticle configuration to define 
```

Comment:
why it can be named deliveryOptions ??

Reply:
We want to be able to specify deliveryOptions for a repository connector and knots separately.

Reply:
If we want to have separate delivery options per each connected address, maybe we should  have "global"  `deliveryOptions` that defines default options for each Proxy and additionally allow to define options inside each section and keep consistent name `deliveryOptions`, e.g.:

```json
{
  "main": "io.knotx.server.KnotxServerVerticle",
  "options": {
    "config": {
      "httpPort": 8092,
      "displayExceptionDetails": true,
      "deliveryOptions": {
        "timeout": 15000
      },
      "repositories": [
        {
          "path": "/content/local/.*",
          "address": "knotx.core.repository.filesystem",
          "deliveryOptions": {
            "timeout": 30000
          }
        },
        {
          "path": "/content/.*",
          "address": "knotx.core.repository.http",
          "deliveryOptions": {
            "timeout": 30000
          }
        }
      ],
      "splitter": {
        "address": "knotx.core.splitter"
      },
      "routing": {
        "GET": [
          {
            "path": ".*",
            "address": "knotx.knot.service",
            "deliveryOptions": {
              "timeout": 60000
            },
            "onTransition": {
              "next": {
                "address": "knotx.knot.handlebars"
              }
            }
          }
        ]
      },
      "assembler": {
        "address": "knotx.core.assembler"
      }
    }
  }
}
```

In the case above, following timeouts will be like:
- filesystem repository: 30s,
- http repository: 30s,
- knotx.knot.service 60s,
- all others 15s

Reply:
@Skejven Your proposal looks very interesting. I recommend to merge the current solution and investige more advanced possibilities if required in the future.

Reply:
Let's then get rid of `repositoryDeliveryOptions` and enable setting it with name `deliveryOptions` either on "globally" on verticle level (e.g. `Server`) or only on producer-level (e.g. only `repositories`).